auth/auth-server/src/server/handle_external_match.rs

```diff
@@ -66,15 +66,22 @@ impl Server {
         body: Bytes,
     ) -> Result<impl Reply, Rejection> {
         // Authorize the request
-        self.authorize_request(path.as_str(), &headers, &body).await?;
+        let key_description = self.authorize_request(path.as_str(), &headers, &body).await?;
 
         // Send the request to the relayer
-        let resp = self.send_admin_request(Method::POST, path.as_str(), headers, body).await?;
+        let resp =
+            self.send_admin_request(Method::POST, path.as_str(), headers, body.clone()).await?;
 
         // Log the bundle parameters
         if let Err(e) = self.log_bundle(resp.body()) {
             warn!("Error logging bundle: {e}");
         }
+
+        // Record metrics
+        if let Err(e) = record_external_match_metrics(&body, resp.body(), key_description).await {
+            warn!("Error recording metrics: {e}");
+        }
+
```

Comment:
Should we spawn metric recording into the background using Tokio to prevent added latency?

Reply:
yeah probably best, just so we don't have to worry about it, you'll want `spawn_blocking` because it looks like all the metrics recorder implementations use a blocking IPC